### PR TITLE
Wallet func

### PR DESCRIPTION
**Pull request by yuanshanhshan**
1. format code
2. rm event db operation
3. abi read path

### CODE REVIEW

File: internal/biz/reportTeeAttestation.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package biz
+
+import (
+	"context"
+	"github.com/go-kratos/kratos/v2/log"
+)
```
Comment (xiaoxiaff) [inline internal/biz/reportTeeAttestation.go:6]: small nit: grouping

File: internal/data/reportTeeAttestationEvent.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+func (t *ReportTeeAttestationEventRepo) FindReportTeeAttestationEvent(ctx context.Context, hash string, txIndex int) (*biz.ReportTeeAttestationEvent, *gorm.DB) {
+	var event *biz.ReportTeeAttestationEvent
+	result := t.data.db.Find(&event, "tx_hash = ? AND tx_index = ?", hash, txIndex)
+	return event, result
+}
+
+func (t *ReportTeeAttestationEventRepo) LastReportTeeAttestationEvent(rtae *biz.ReportTeeAttestationEvent) (*biz.ReportTeeAttestationEvent, error) {
+
```
[the remark this answers is not in the document]
Comment (yuanshanhshan) [reply internal/data/reportTeeAttestationEvent.go:37]: removed

File: internal/worker/chain.go [hunk 3]
```diff
@@ -4,15 +4,14 @@ import (
 	"context"
 	"crypto/ecdsa"
 	"encoding/hex"
+	"fmt"
+	"github.com/carv-protocol/verifier/pkg/attestion"
```
[the remark this answers is not in the document]
Comment (yuanshanhshan) [reply internal/worker/chain.go:8]: resolved

File: internal/worker/chain.go [hunk 4]
```diff
@@ -292,12 +306,29 @@ func (c *Chain) verifyAttestation(ctx context.Context, attestationIds [][32]byte
 
 	tx, err := c.contractObj.VerifyAttestationBatch(auth, attestationIds, results)
 	if err != nil {
-		return errors.Wrap(err, "contract VerifyAttestationBatch error")
+		fmt.Println("txError: ", err)
```
[the remark this answers is not in the document]
Comment (yuanshanhshan) [reply internal/worker/chain.go:309]: removed

File: internal/biz/reportTeeAttestation.go (hunk 1 above)
Comment (yuanshanhshan) [reply internal/biz/reportTeeAttestation.go:6]: resolved

**Pull request merged**